homeassistant/components/squeezebox/strings.json

```
@@ -5,6 +5,9 @@
       "user": {
         "data": {
           "host": "[%key:common::config_flow::data::host%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of your Squeezebox."
```

```suggestion
          "host": "The hostname or IP address of your Logitech Media Server."
```
Suggest a minor tweak, as the integration talks to a "Logitech Media Server" rather than the squeezebox itself.

---

Thank you!